src/common/gui/main-styles.ts

```diff
@@ -2641,5 +2641,13 @@ styles.registerStyle("main", () => {
 		".single-line-text-field::placeholder": {
 			color: theme.content_message_bg,
 		},
+		".tutaui-button-outline": {
```

Comment:
we also try to not add classes for components and use utility classes instead if we can help it